modules/orca/output.tf

```diff
@@ -1,20 +1,65 @@
-output "orca_rds" {
-  value = module.orca_rds.rds
+## Lambda Module Outputs (orca_lambda)
+## =============================================================================
+# Ingest Lambdas
+# ------------------------------------------------------------------------------
+output "orca_lambda_copy_to_glacier_arn" {
+  description = "AWS ARN of the ORCA copy_to_glacier lambda."
+  value       = module.orca_lambdas.copy_to_glacier_arn
 }
 
-output "orca_subnet_group" {
-  value = module.orca_rds.rds_subnet_group
-}
 
-output "orca_lambda_extract_filepath_arn" {
-  value = module.orca_lambdas.extract_filepath_arn
+# Recovery Lambdas
+# ------------------------------------------------------------------------------
+output "orca_lambda_extract_filepaths_for_granule_arn" {
+  description = "AWS ARN of the ORCA extract_filepaths_for_granule lambda."
+  value       = module.orca_lambdas.extract_filepaths_for_granule_arn
 }
 
+
 output "orca_lambda_request_files_arn" {
-  value = module.orca_lambdas.request_files_arn
+  description = "AWS ARN of the ORCA request_files lambda."
+  value       = module.orca_lambdas.request_files_arn
 }
 
-output "orca_lambda_copy_to_glacier_arn" {
-  value = module.orca_lambdas.copy_to_glacier_arn
+
+output "orca_lambda_copy_files_to_archive_arn" {
+  description = "AWS ARN of the ORCA copy_files_to_archive lambda."
+  value       = module.orca_lambdas.copy_files_to_archive_arn
+}
+
+
+output "orca_lambda_request_status_arn" {
+  description = "AWS ARN of the ORCA request_status lambda."
+  value       = module.orca_lambdas.request_status_arn
+}
+
+
+output "orca_lambda_request_status_for_granule_arn" {
+  description = "AWS ARN of the ORCA request_status_for_granule lambda."
+  value       = module.orca_lambdas.request_status_for_granule_arn
+}
+
+
+output "orca_lambda_request_status_for_job_arn" {
+  description = "AWS ARN of the ORCA request_status_for_job lambda."
+  value       = module.orca_lambdas.request_status_for_job_arn
+}
+
+
+## Workflow Module Outputs (orca_workflows)
+## =============================================================================
+## No workflow outputs currently requested/needed
+
+
+## RDS Module Outputs (orca_rds)
+## =============================================================================
+output "orca_rds" {
```

Comment:
this output prints all the parameters of orca_rds resource. Just making sure if this is intentional. I guess the RDS is going away so it's not a big deal.

Reply:
Yes, this was unchanged from the original output.tf file. I just added a description. From what I could tell, this is not used anywhere and my best guess is it was used for some verification on the development side. I left it in for the time being knowing that it will likely go away soon with the module.